tests/unit/amo/components/TestInstallWarning.js

```diff
@@ -98,29 +95,20 @@ describe(__filename, () => {
       expect(component.instance().couldShowWarning()).toEqual(true);
     });
 
-    it.each([UNINSTALLED, UNKNOWN])(
-      `returns true if mozAddonManager exists and the add-on has a status of %s`,
+    it.each(validInstallStates)(
```

Comment:
We don't really need this test anymore, as any code that looks at install status has been removed. Therefore, there doesn't seem to be a reason for a test that makes sure that this returns `true` for a number of different statuses.

On the other hand, a code change was made that means that, whereas a number of statuses (including `INSTALLED`) in the past would have had this return `false`, it will now return `true` in those cases. So, do we need some sort of test that verifies this new behaviour?

Reply:
Is the answer to this question to just add a comment above this test saying:

`// See https://github.com/mozilla/addons-frontend/issues/8642.`

(Which I just added)

Reply:
I would remove the test case because you would need a big change (aka something that will not pass a review) to make it fail.